app/helpers/view_components/view_component.rb

```diff
@@ -1,6 +1,14 @@
 module ViewComponents
   class ViewComponent
     include ActionView::Helpers::TagHelper
+    include ActionView::Helpers::UrlHelper
+    include IconsHelper
+
+    # TODO: This can probably be removed
+    include ActionView::Helpers::AssetTagHelper
+
+    include ActionView::Context
+
     extend Mandate::Memoize
```

Comment:
I think it would be worth splitting this into two classes `ReactComponent` and `RenderedComponent`, as they require different `includes` etc. I'll do this in another PR.